app/controllers/admin/api/services/proxies_controller.rb

```diff
@@ -76,7 +76,7 @@ def proxy_params
                           error_status_auth_failed error_status_auth_failed error_headers_auth_failed
                           error_status_auth_missing error_headers_auth_missing error_no_match
                           error_status_no_match error_headers_no_match secret_token hostname_rewrite
-                          oauth_login_url api_test_path oidc_issuer_endpoint]
+                          oauth_login_url api_test_path oidc_issuer_endpoint oidc_issuer_type]
```

Comment:
We should add it to the documentation https://github.com/3scale/porta/blob/fc6725093bdbf37558e39484b42f58eaa8feeaff/app/controllers/admin/api/services/proxies_controller.rb#L56